### PR TITLE
Add web changes to What's New

### PR DESCRIPTION
**Pull request by manics**
- Updated sysadmin What's New page with web config changes.
- Mentioned web `clearsessions` in Server Upgrade page.

--no-rebase

### CODE REVIEW

File: omero/sysadmins/server-upgrade.txt [hunk 1]
```diff
@@ -241,23 +241,18 @@ Your memory settings should be copied along with :file:`etc/grid/config.xml`,
...
-As a result of this your web server configuration stanza generated by
-the previous version of ``bin/omero web config`` has to be replaced with
-the new version. To generate the relevant configuration, please run
-``bin/omero web config <webserver>``, update and restart your web
-server.
+OMERO.web server configuration
+""""""""""""""""""""""""""""""
+
+The generated web-server configurations for Nginx and Apache have been
+revised in OMERO 5.1. It is highly recommend that you :ref:`regenerate your
```
Comment (mtbc) [inline omero/sysadmins/server-upgrade.txt]: recommended

File: omero/sysadmins/server-upgrade.txt [hunk 2]
```diff
@@ -241,23 +241,18 @@ Your memory settings should be copied along with :file:`etc/grid/config.xml`,
...
+configuration <fastcgi_configuration>`, remembering to merge in any of your
+own modifications if necessary. See :doc:`/sysadmins/whatsnew` for details
+of changes.
+
+If necessary ensure you have setup a regular task to clear out any stale
```
Comment (mtbc) [inline omero/sysadmins/server-upgrade.txt]: "set up" (setup is noun)

**Comment by manics** [comment]
@mtbc Thanks, now fixed

**Comment by jburel** [comment]
in server upgrade:
 * it is suggested to send the output of the ```bin/omero admin diagnostics``` to ome-users. Probably more suitable to send it to devel.

**Comment by manics** [comment]
@jburel I've removed the double space. I think `ome-users` is the correct list because it's to do with configuring/using a server, not code changes.

**Comment by jburel** [comment]
Not convinced that a general user will run the command, the most important point is that we are notified and people get help.

**Comment by manics** [comment]
I thought users includes sysadmins who deploy the server, with devel reserved for development questions?

**Comment by hflynn** [comment]
For the purposes of feedback, I was also under the assumption that sysadmins are treated as users not developers. It is certainly my understanding that many people running the server are not on the devel list.

**Comment by jburel** [comment]
works for me!

**Comment by mtbc** [comment]
I normally check both lists anyway in preparing Community for stand-up.

**Comment by jburel** [comment]
Good to go

**Pull request merged**